src/Services/SmartProxy.js

```diff
@@ -26,6 +26,11 @@ export const SmartProxyApi = createApi({
     getClusters: builder.query({
       query: () => `v2/clusters`,
     }),
+    getDisplayName: builder.query({
```

Comment:
why create an extra query? you can utilize `getClusterById`, since the response already contains the required display name value

Reply:
Yes, I see. But you will make **two** identical requests. And what is a problem with just making a selector to the data already fetched and saved at the redux store?

Reply:
Hm! That's an idea, thanks

Reply:
@gkarat PTAL